LedgerHQ/ledger-live-desktop
# feat(walletconnect): extends compatible networks for the auto flow

## 🦒 Context (issues, jira)

During the automatic authentification flow, the user can now choose a
Polygon or BSC account.

## 💻  Description / Demo (image or video)



## 🖤  Expectations to reach

PR must pass CI, rebase develop if conflicts. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
- **on delivery**: at least one of these two checkboxes must be checked: 



## 🧪 How to test

- Ensure you have BSC/Polygon account
- Build the app by running `yarn dist`
- Launch the application by running the output of the dist command. (macos: dist/mac/Ledger Live.app)
- Go to [Paraswap](https://paraswap.io/) dApp (or any dApps that supports walletconnect) and connect using walletconnet
- At this step, the walletconnect modal must be displayed in the Ledger Live app and BSC/Polygon accounts should be selectable from the select currency element
